src/components/experiments/wizard/MinDiffCalculator.tsx

```diff
@@ -0,0 +1,378 @@
+              <StepLabel>What is the baseline monthly revenue?</StepLabel>
+              <StepContent>
+                <TextField
+                  id={`baseline-revenue-${calculatorUniqueId}`}
+                  value={metricBaselineRevenue}
+                  onChange={onChangeMetricBaselineRevenue}
+                  label='Baseline revenue / month'
+                  placeholder='9800'
+                  helperText={`= ${roundToNumPlaces(nanToZero(metricBaselineRevenuePerUser), 4)}$ ARPU`}
+                  variant='outlined'
+                  fullWidth
+                  required
+                  type='number'
+                  inputProps={{
+                    min: 0,
+                    max: 100,
+                  }}
+                  InputLabelProps={{
+                    shrink: true,
+                  }}
+                  InputProps={{
+                    endAdornment: (
+                      <InputAdornment position='end' className={classes.adornment}>
+                        USD
+                      </InputAdornment>
+                    ),
+                  }}
+                />
+              </StepContent>
+            </Step>
+          )}
+          {isConversion ? (
+            <Step active key={'conversion-3'}>
+              <StepLabel>How many extra conversions makes a practical difference?</StepLabel>
+              <StepContent>
+                <TextField
+                  id={`extra-conversions-per-month-${calculatorUniqueId}`}
+                  value={metricConversionsPerMonthMinPracticalDiff}
+                  onChange={onChangeMetricConversionsPerMonthMinPracticalDiff}
+                  label='Extra conversions / month'
+                  placeholder='100'
+                  variant='outlined'
+                  fullWidth
+                  required
+                  type='number'
+                  helperText={`= ${roundToNumPlaces(nanToZero(metricMinimumPracticalDifferenceLift), 2)}% lift`}
+                  inputProps={{
+                    min: 0,
+                  }}
+                  InputLabelProps={{
+                    shrink: true,
+                  }}
+                />
+              </StepContent>
+            </Step>
+          ) : (
+            <Step active key={'revenue-3'}>
+              <StepLabel>How much extra monthly revenue makes a practical difference?</StepLabel>
+              <StepContent>
+                <TextField
+                  id={`extra-revenue-${calculatorUniqueId}`}
+                  value={metricRevenuePerMonthMinPracticalDiff}
+                  onChange={onChangeMetricRevenuePerMonthMinPracticalDiff}
+                  label='Extra revenue / month'
+                  placeholder='1500'
+                  variant='outlined'
+                  fullWidth
+                  required
+                  type='number'
+                  helperText={`= ${roundToNumPlaces(nanToZero(metricMinimumPracticalDifferenceLift), 4)}% lift`}
```

Comment:
```suggestion
                  helperText={`= ${roundToNumPlaces(nanToZero(metricMinimumPracticalDifferenceLift), 2)}% lift`}
```
2 decimals for the Revenue lift should be more than enough 😁